p4-fusion/main.cc

```diff
@@ -155,7 +155,8 @@ int Main(int argc, char** argv)
 	// Return early if we have no work to do
 	if (changes.empty())
 	{
-		SUCCESS("Repository is up to date. Exiting.")
+		SUCCESS("Repository is up to date. Updating tags.")
+		updateTags(&p4, depotPath, git.GetRepoPtr());
```

Comment:
I wonder if we should make the tags thing an optional feature? Via a cmd flag or so.